examples/src/flat/flat.vert

```diff
@@ -0,0 +1,20 @@
+#version 450 core
+#extension GL_ARB_separate_shader_objects : enable
```

Comment:
Hi there, just passing by :D
With `450 core`, enabling the extension basically does nothing, as 450 requires this to be a supported extension. If you were using versions lower than 410, `enable` could do something; but I'm not sure if you'd really want to use that.
IMO, you *should* be using `require` so the shader compilation can fail gracefully if your requested extension is not available.